Units/parser-openapi.r/swagger.d/input.yaml

```diff
@@ -0,0 +1,64 @@
+swagger: '2.0'
+info:
+    description: test
+    title: test
```

Comment:
How about tagging the title?

Reply:
If it makes sense to tag all user-defined entities, then ok.
But then more fields must be tagged:
- info itself
- servers children
- webhooks children
- security children
- tags children

I don't think the following entities worth adding tags (too minor IMHO):
- openapi
- jsonSchemaDialect
- externalDocs

https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#openapi-object

Reply:
Also the items of "components": https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#fixed-fields-6

Reply:
added

Reply:
The way I wrote is wrong.
In 97915e70b75ca702b916278907d10476394b516d, your parser extracts "title".
However, what I expected is your parser extract "test" in "title: test".